src/screens/track-screen/TrackScreenHeader.tsx

```diff
@@ -421,14 +421,14 @@ export const TrackScreenHeader = ({
       // userId={coSign.user.user_id}
     >
       <DynamicImage
-        image={{ uri: image }}
-        style={[styles.coverArt] as ImageStyle[]}
+        source={{ uri: image }}
```

Comment:
thanks for the fix, weird didnt catch this with typecheck earlier...

Reply:
Idk why it didn't catch, but it was my fault bc I merged track screen after the dynamic image change

Reply:
ah, makes sense